ScanCommand/src/main/java/io/ballerina/scan/ScanCmd.java

```diff
@@ -114,8 +121,38 @@ public void execute() {
             return;
         }
 
+        Project project = getProject();
+        if (project == null) {
+            return;
+        }
+
+        // Initialize project analyzer
+        ProjectAnalyzer projectAnalyzer = new ProjectAnalyzer();
+
         outputStream.println();
         outputStream.println("Running Scans...");
+
+        // Perform core scans
+        List<Issue> issues = projectAnalyzer.analyzeProject(project);
+
+        if (platforms.isEmpty() && !platformTriggered) {
+            // Print issues to console
+            ScanUtils.printToConsole(issues, outputStream);
+
+            if (project.kind().equals(ProjectKind.BUILD_PROJECT)) {
+                // Save results to directory
+                Path reportPath;
+                if (targetDir != null) {
+                    reportPath = ScanUtils.saveToDirectory(issues, project, targetDir);
+                } else {
+                    reportPath = ScanUtils.saveToDirectory(issues, project, null);
+                }
+
+                outputStream.println();
+                outputStream.println("View scan results at:");
```

Comment:
This can be same line with the report path, wdyt?

Reply:
- Having the report path provided in a new line promotes readability similar to the [testerina coverage report](https://github.com/ballerina-platform/ballerina-lang/blob/master/cli/ballerina-cli/src/main/java/io/ballerina/cli/utils/TestUtils.java#L152).

Reply:
+1